protobuf/video/sfu/event/events.proto

```diff
@@ -83,18 +82,16 @@ message TrackUnpublished {
   models.Participant participant = 5;
 }
 
+// JoinRequest is sent by the participant to join a call. The
+// call ID of the call should be encoded in the auth token.
 message JoinRequest {
   string token = 1;
   string session_id = 2;
-  // dumb SDP that allow us to extract subscriber's decode codecs
-  string subscriber_sdp = 3;
```

Comment:
How would the SFU determine the decode capabilities of the clients if we remove this? 
Did something change in the meantime?

Reply:
Well...We currently don't use this field in the SFU. It has been like that for quite some time now. @thesyncim - any plan to use it in the future?

Reply:
Spoke to @thesyncim. I'll be restoring the field.

Reply:
Awesome! Thank you for the clarification 👍